msi.sh

```diff
@@ -0,0 +1,11 @@
+NAME="SecretHub CLI"
+EXE=bin/secrethub.exe
+VERSION=$(git describe --tags $(git rev-list --tags --max-count=1))
+MANUFACTURER=SecretHub
+UPGRADE_CODE=7178CB18-B431-4A9D-B29E-D31E63A1CF77
```

Comment:
Is this a static value?

Reply:
Yes this is an arbitrary GUID to identify our app, so Windows knows it's same app across potential name changes.

Reply:
Ah ok, got it 👍 Consider adding a comment as this is some weird arcane MSI knowledge.